src/main/resources/assets/tfc/lang/en_us.json

```diff
@@ -216,6 +216,38 @@
   "tfc.tooltip.pot_boiling": "Boiling!",
   "tfc.tooltip.pot_finished": "Finished",
   "tfc.tooltip.pot_ready": "Ready",
+  "tfc.jade.growth": "Growth: ",
+  "tfc.jade.progress": "Progress: ",
+  "tfc.jade.sealed_date": "Sealed Date: ",
```

Comment:
As in #2086, these should be tooltips like `"Growth: %s"`, `"Progress: %s"` as it gives syntactic control to the language in question.